python/vyos/util.py

```diff
@@ -182,16 +182,20 @@ def call(command, flag='', shell=None, input=None, timeout=None, env=None,
     return code
 
 
-def read_file(fname, defaultonfailure=None):
+def read_file(fname, defaultonfailure=None, strip_end=True):
```

Comment:
Is is bad practice to change global utility functions in a feature PR. Please remove this change from the PR. Also I guess this is no longer necessary as the configuration should be written as once and not read, modified and written back again.

Reply:
Removed this change. Most likely, this can lead to the unnecessary triggering of the `daemons` file updates (we have a newline in both templates and default files), but from the first view, this should not be a problem and may be fixed back in additional PR for `util.py`.